lib/db/schema/patch-062-063.sql

```diff
@@ -0,0 +1,155 @@
+ALTER TABLE devices
+ADD COLUMN pushEndpointExpired BOOLEAN NOT NULL DEFAULT FALSE AFTER callbackAuthKey,
```

Comment:
Can we make the name more homgeneous with the other push-related columns? Since they're all `callback...`, maybe `callbackIsExpired`, which also has the benefit of sounding more booleany?